app/components/unranked_suppliers/details_component.html.haml

```diff
@@ -2,8 +2,12 @@
   .unranked-supplier__title{ "data-testid": "unranked-supplier-details-#{supplier.name.parameterize}" }
     = title
   - if Feature.enabled? "FF_SMALL_SUPPLIER_STARS"
-    %p
-      = whitelabel_supplier_link
+    -if whitelabelled?
+      %p
+        = whitelabel_supplier_link
+    -else
+      -if star_ratings?
+        = render UnrankedSuppliers::StarsSummaryComponent.new(supplier)
```

Comment:
There is a [`render?`](https://viewcomponent.org/guide/conditional_rendering.html) hook in view components that is for this purpose - so it would be good to move the `star_ratings?` logic into the `StarsSummaryComponent` (you're passing it the `supplier` object anyway)

Reply:
Very good point - thanks - I have moved it :)